src/plugins/jartic/archive_downloader.py

```
@@ -0,0 +1,241 @@
+import logging
+import re
+import time
+import zipfile
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Dict, List, Optional
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)